src/main/java/Storage.java

```diff
@@ -0,0 +1,69 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Scanner;
```

Comment:
Maybe you can separate imports from different packages so that it will be easier to browse the list?